src/main/io/vtx_tramp.c

```diff
@@ -118,196 +150,107 @@ static uint8_t trampChecksum(uint8_t *trampBuf)
     return cksum;
 }
 
-static bool trampVtxControlEnabled(void)
-{
-    return !IS_RC_MODE_ACTIVE(BOXVTXCONTROLDISABLE);
-}
-
+// Check if race lock is enabled
 static bool trampVtxRaceLockEnabled(void)
 {
-    return trampControlMode & TRAMP_CONTROL_RACE_LOCK;
+    return trampCurControlMode & TRAMP_CONTROL_RACE_LOCK;
 }
 
+// Send tramp protocol frame to device
 static void trampSendU16(uint8_t cmd, uint16_t param)
 {
     if (!trampSerialPort) {
         return;
     }
 
     memset(trampReqBuffer, 0, ARRAYLEN(trampReqBuffer));
-    trampReqBuffer[0] = 15;
+    trampReqBuffer[0] = 0x0F;
     trampReqBuffer[1] = cmd;
     trampReqBuffer[2] = param & 0xff;
     trampReqBuffer[3] = (param >> 8) & 0xff;
     trampReqBuffer[14] = trampChecksum(trampReqBuffer);
-    trampWriteBuf(trampReqBuffer);
+    serialWriteBuf(trampSerialPort, trampReqBuffer, 16);
 }
 
+// Send command to device
 static void trampSendCommand(uint8_t cmd, uint16_t param)
 {
-    if (trampVtxControlEnabled()) {
+    // Is VTX control enabled?
+    if (!IS_RC_MODE_ACTIVE(BOXVTXCONTROLDISABLE)) {
+        // Yes, send command
         trampSendU16(cmd, param);
     }
 }
 
-static bool trampValidateFreq(uint16_t freq)
-{
-    if (trampRFFreqMin != 0 && trampRFFreqMax != 0) {
-        return (freq >= trampRFFreqMin && freq <= trampRFFreqMax);
-    } else {
-        return (freq >= VTX_TRAMP_MIN_FREQUENCY_MHZ && freq <= VTX_TRAMP_MAX_FREQUENCY_MHZ);
-    }
-}
-
-static void trampDevSetFreq(uint16_t freq)
-{
-    trampConfFreq = freq;
-    if (trampConfFreq != trampCurFreq) {
-        trampFreqRetries = TRAMP_MAX_RETRIES;
-    }
-}
-
-void trampSetFreq(uint16_t freq)
-{
-    trampDevSetFreq(freq);
-}
-
-void trampSendFreq(uint16_t freq)
-{
-    if (!trampVtxRaceLockEnabled()) {
-        trampSendCommand('F', freq);
-    }
-}
-
-void trampSetRFPower(uint16_t level)
-{
-    trampConfPower = level;
-    if (trampConfPower != trampConfiguredPower) {
-        trampPowerRetries = TRAMP_MAX_RETRIES;
-    }
-}
-
-void trampSendRFPower(uint16_t level)
-{
-    if (!trampVtxRaceLockEnabled()) {
-        trampSendCommand('P', level);
-    }
-}
-
-// return false if error
-bool trampCommitChanges(void)
-{
-    if (trampStatus != TRAMP_STATUS_ONLINE) {
-        return false;
-    }
-
-    trampStatus = TRAMP_STATUS_SET_FREQ_PW;
-    return true;
-}
-
-void trampSetPitMode(uint8_t onoff)
-{
-    trampSendCommand('I', onoff ? 0 : 1);
-}
-
-// returns completed response code
+// Process response and return code if valid else 0
 static char trampHandleResponse(void)
 {
     const uint8_t respCode = trampRespBuffer[1];
 
     switch (respCode) {
-    case 'r':
+        case 'r':
```

Comment:
Betaflight coding style has the `case` statements at the same indentation level as the `switch` statement.

Reply:
My bad, all switch statements updated to follow the standard. I've left the block's in though, the coding standard doesn't mention them directly. Are they ok to stay?

Reply:
There is no hard requirement in the coding standard for blocks inside switch, so this is fine.